Apps/Admin/Tests/Functional/cypress/integration/e2e/patient-details.cy.js

```diff
@@ -59,6 +59,10 @@ describe("Patient details message verification", () => {
             .click();
 
         cy.get("[data-testid=block-access-switches]").should("be.visible");
+        cy.get(`[data-testid*="block-access-switch"]`).should(
+            "not.have.attr",
+            "readonly"
+        );
         cy.get("[data-testid=block-access-cancel]").should("not.exist");
```

Comment:
After checking if line 62 does not contain read only, a dataset needs to be selected before checking if the save and cancel buttons exist.  As an admin, the save and cancel buttons should exist.